cdr/resolver.go

```diff
@@ -187,22 +187,17 @@ func marshalCanonicalCAASet(set []*dns.CAA) ([]byte, error) {
 func (cdr *CAADistributedResolver) LookupCAA(ctx context.Context, domain string) ([]*dns.CAA, error) {
 	query := make(url.Values)
 	query.Add("name", domain)
-	query.Add("type", strconv.Itoa(int(dns.TypeCAA))) // CAA
-	rawQuery := query.Encode()
+	query.Add("type", strconv.Itoa(int(dns.TypeCAA)))
+	url := fmt.Sprintf("%s?%s", cdr.URI, query.Encode())
```

Comment:
Alternately let's back out this diff unless you can reproduce or remember the problem that it's fixing.